trillian_map_api.proto

```diff
@@ -108,6 +108,11 @@ service TrillianMap {
   // For indexes that do not exist, the inclusion proof will use nil for the empty leaf value.
   rpc GetLeaves(GetMapLeavesRequest) returns(GetMapLeavesResponse) {}
   rpc GetLeavesByRevision(GetMapLeavesByRevisionRequest) returns(GetMapLeavesResponse) {}
+  // SetLeaves sets the values for the provided leaves, and returns the new map root if successful.
```

Comment:
Currently, I think it's tighter than that - the identical set of leaves must be retried. 
(Because partial writes of subtrees at rev n+1 are never deleted; if you set a superset of leaves then some of these subtrees [particularly up at the top of the tree] will need to be updated and there's no support for that right now.)

Reply:
Made the text tighter.